docs/install/deb.md

```diff
@@ -42,11 +42,20 @@ These steps assume you're using Ubuntu 18.04.
 
 1. Install Elasticsearch OSS:
 
+   Install x64
+
    ```bash
    wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-{{site.es_version}}-amd64.deb
    sudo dpkg -i elasticsearch-oss-{{site.es_version}}-amd64.deb
    ```
 
+   Install ARM64
+
+   ```bash
+   wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-{{site.es_version}}-arm64.deb
+   sudo dpkg -i elasticsearch-oss-{{site.es_version}}-arm64.deb
+   
```

Comment:
Missing the closing backticks here. For general formatting, I'd prefer a single code block with both options, similar to how we present the Java 11 vs. Java 8 step here: https://opendistro.github.io/for-elasticsearch-docs/docs/install/rpm/

```
# x86
wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-{{site.es_version}}-amd64.deb
sudo dpkg -i elasticsearch-oss-{{site.es_version}}-amd64.deb
# ARM64
wget https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-oss-{{site.es_version}}-arm64.deb
sudo dpkg -i elasticsearch-oss-{{site.es_version}}-arm64.deb
```

Reply:
I have made the change accordingly.